### ISSUE TITLE
Color contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by rbruhn** [AustinMastLab/iDigInfo-Mastlab-D7#7]
Description
There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.

**Comment by rbruhn** [comment]
Fixed with commit 0fa2d58ef4933bab119dab6bb2c789db94cbd824
